pkg/metrics/client_metrics.go

```diff
@@ -0,0 +1,34 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"net/url"
+	"time"
+
+	clientmetrics "k8s.io/client-go/tools/metrics"
+	_ "sigs.k8s.io/controller-runtime/pkg/metrics" // Needed for init() side effect
+)
+
+// DisableRESTClientMetrics disables the rest client latency histograms configured by
+// controller-runtime in sigs.k8s.io/controller-runtime/pkg/metrics/client_go_adapter.go#registerClientMetrics
+func DisableRESTClientMetrics() {
```

Comment:
Should this be called somewhere?  Perhaps it's obvious and I'm not seeing it.

Reply:
It's [here](https://github.com/open-policy-agent/gatekeeper/pull/1482/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R160) in `main.go`.